tools/Google.Cloud.Tools.Analyzers.Tests/Helpers/DiagnosticVerifier.Helper.cs

```diff
@@ -0,0 +1,173 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Text;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace TestHelper
```

Comment:
I've just looked, and a lot of this code exists elsewhere... can we pull in an existing nuget package instead of repeating things? If not, we'll need to find out what the process is for including code from elsewhere in our own repo. Using a submodule may be a better approach.